.evergreen/ssdlc-compliance-report-template.md

```diff
@@ -0,0 +1,32 @@
+# MongoDB Rust Driver SSDLC Compliance Report
+
+### Release Version: RELEASE_VERSION
+
+**Release Creator**
+The creator of this release can be determined by visiting
+https://github.com/mongodb/mongo-rust-driver/releases/tag/vRELEASE_VERSION.
+
+**Process Document**
+TODO RUST-1918 Link to "How We Develop Software" document
+
+**Tool used to track third party vulnerabilities**
+N/A; the Rust driver does not bundle third-party dependencies
+
+**Third-Party Dependency Information**
+N/A; the Rust driver does not bundle third-party dependencies
+
+**Static Analysis Findings**
+TODO RUST-1920 Link to static analysis report
```

Comment:
Since the static analysis report will only be provided to approved partners upon request, and not generally shared with customers, I think this will need to change to something like "To request a copy of the static analysis report, please contact driver-rust-team@10gen.com."

Reply:
sounds good, going to leave out our email to avoid linking to it in the repo :) I think customers requesting this will already have contacts to reach out to